go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/ubuntu/adsys
 
-go 1.22.0
-
-toolchain go1.22.7
+go 1.22.7
```

Comment:
Do we really rely on 1.22.7 from a feature or security vulnerability perspective?

Reply:
As discussed in call: Although we do rely on 1.22.7 to bump some dependencies, the current available Go version in Jammy is still 1.22.2, so we can't go (see what I did here? :joy:) beyond that.